validations/management/commands/update_validations.py

```
@@ -0,0 +1,189 @@
+        jsonschema.validate(instance=data, schema=schema)
+
+    def _validate_validations_exist(self, updates: List[Dict[str, Any]]) -> None:
+        """Validate that all referenced validations exist in the database"""
+        for update in updates:
+            screen_uuid = update["screen_uuid"]
+            program_name = update["program_name"]
+
+            try:
+                screen = Screen.objects.get(uuid=screen_uuid)
+            except Screen.DoesNotExist:
+                raise CommandError(f"Screen with UUID '{screen_uuid}' does not exist")
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)